src/main/java/net/starschema/clouddb/jdbc/BQForwardOnlyResultSet.java

```diff
@@ -90,6 +93,10 @@
      * The -1 is needed because of the while(Result.next() == true) { } iterating method*/
     private int Cursor = -1;
 
+    private final DateTimeFormatter TIMESTAMP_FORMATTER = DateTimeFormatter
+            .ofPattern("yyyy-MM-dd HH:mm:ss")
```

Comment:
Do we not want this to have any extra precision here? I know other jdbc drivers have fractional seconds in their base implementation. I don't know if there is any standard way of doing this though.